app/backend/approaches/retrievethenreadvision.py

```diff
@@ -83,7 +85,7 @@ async def run(
         minimum_reranker_score = overrides.get("minimum_reranker_score", 0.0)
         filter = self.build_filter(overrides, auth_claims)
 
-        vector_fields = overrides.get("vector_fields", ["embedding"])
+        vector_fields = overrides.get("vector_fields", [self.embedding_field])
```

Comment:
Similar here, need to decide whether the frontend should render the actual field names.

Reply:
No need

Reply:
just something vague like "vectors"